docs/guide/custom_policies.md

```diff
@@ -163,22 +291,76 @@ ezplatform:
     system:
         default:
             limitation_value_templates:
-                - { template: limitation/custom_limitation_value.html.twig, priority: 0 }
-
+                - { template: '@ezdesign/limitation/custom_limitation_value.html.twig', priority: 0 }
 ```
 
-!!! note
+To check if current user has this custom limitation set to true from a custom controller:
+```php
+<?php
 
-    If you skip this part, Limitation values will be rendered using an [`ez_limitation_value_fallback`](https://github.com/ezsystems/repository-forms/blob/master/bundle/Resources/views/limitation_values.html.twig#L1-L6) block as comma-separated list.
```

Comment:
Have a better look to this. Maybe useful when using `MultipleSelectionBasedMapper`.

Reply:
https://github.com/ezsystems/repository-forms/blob/2.5/bundle/Resources/views/limitation_values.html.twig#L1
TODO: try if this fallback is used when there is no CustomLimitationValueMapper service.

Reply:
The fallback is used when the CustomLimitationValueMapper service is commented out.